map/framework.cpp

```diff
@@ -2250,7 +2250,7 @@ place_page::Info Framework::BuildPlacePageInfo(place_page::BuildInfo const & bui
     {
       auto const & track = *GetBookmarkManager().GetTrack(buildInfo.m_trackId);
       auto rect = track.GetLimitRect();
-      track.UpdateSelectionInfo(track.GetLimitRect(), trackSelectionInfo);
+      track.UpdateSelectionInfo(rect, trackSelectionInfo);
```

Comment:
Better fix is to remove "auto rect = ..." line.

Reply:
Done